apiserver/facades/client/application/deploy.go

```diff
@@ -184,10 +185,21 @@ func DeployApplication(
 			}
```

Comment:
The `DeployApplicationParams` struct still includes a resources field. These resources are passed through from `Deploy` in application.go. These should either be removed or marked as deprecated, it could be misleading in the future to have this field that is never used.